packages/accordion/src/js/jquery.js

```diff
@@ -0,0 +1,86 @@
+/*! [replace-name] v[replace-version] */
+/***************************************************************************************************************************************************************
+ *
+ * Accordion function
+ *
+ * A component to allow users to show or hide page elements.
+ *
+ **************************************************************************************************************************************************************/
+
+
+/**
+ * The accordion jquery function to add attributes and event listeners
+ *
+ * Public functions are:
+ * $('.js-uikit-accordion-wrapper').uikitAccordion().isOpen()
```

Comment:
Our method is build so it runs multiple instances and that's even tested on the jquery test page.

The slightly stinky line 71 is so we avoid adding event handlers twice if a user runs `$('.js-uikit-accordion-wrapper').uikitAccordion().isOpen()` multiple times :). Defensive programming and all. The method can then also be used if the html was added dynamically.